### PR TITLE
[LLM Runtime] Optimize tests of llm runtime

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

others

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: tests/test_llm_runtime.py [hunk 1]
```diff
@@ -28,28 +28,37 @@ def tearDownClass(cls) -> None:
...
         tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
         inputs = tokenizer(prompt, return_tensors="pt")
-
-        # pytorch fp32
-        pt_model = AutoModelForCausalLM.from_pretrained(model_name, trust_remote_code=True)
-        pt_model.eval() 
-        pt_logits = pt_model(input_ids=inputs.input_ids).logits[:,-1]
-        pt_generate_ids = pt_model.generate(input_ids=inputs.input_ids, do_sample=False, max_new_tokens=100)[0].tolist()
+        
+        pt_logits = torch.load("/tf_dataset2/inc-ut/nlptoolkit_ut_model/llama2_pt_logits.pth")[:,-1]
```
Comment (airMeng) [inline tests/test_llm_runtime.py:36]: if this is just for CI, shall we designate the path under .github/

Comment (VincyZhang) [reply tests/test_llm_runtime.py:36]: This is the path inside docker container

Comment (airMeng) [reply tests/test_llm_runtime.py:36]: can we setup an environmental variable for path since users might want to verify these UT?

**Review by a32543254 — APPROVED** [review]
LGTM

File: tests/test_llm_runtime.py (hunk 1 above)
Comment (VincyZhang) [reply tests/test_llm_runtime.py:53]: Users still need to prepare those local file to run UTs no matter setup or not setup a variable
Actually the best way is to judge if you have local file then use local file, if you don't have local file, UT can lead you to generate those. In this way many of our UT need to rewrite.

**Pull request merged**